modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.es.js

```diff
@@ -60,6 +60,14 @@ class FormRenderer extends Component {
 	_handleFieldFocused(event) {
 		this.emit('fieldFocused', event);
 	}
+
+	_getDisplayableFn() {
+		return (
+			this.initialConfig_.readOnly ||
```

Comment:
@linolaoj, from where this `initialConfig_` comes from?

Reply:
@natocesarrego , initialConfig_ holds the values passed when creating the FormRenderer. But I see there is a more clean way to access these values through the "state". I'll change that.